faq/troubleshooting.rst

```diff
@@ -136,3 +136,16 @@ It is possible that operating conan, some random exceptions (some with complete
 - The user has some file or folder open (in a file editor, in the terminal), so it cannot be removed, and the process fails. Make sure to close files, specially if you are opening or inspecting the local conan cache.
 - In Windows, the Search Indexer might be opening and locking the files, producing random, difficult to reproduce and annoying errors. Please **disable the Windows Search Indexer for the conan local storage folder**
 
+
+ERROR: Error while initializing Options
+---------------------------------------
+
+When installing a Conan package and the follow error occurs:
+
+.. code-block:: text
+
+    ERROR: conanfile.py: Error while initializing options. Please define your default_options as list or multiline string
+
+Problably your Conan version is outdated.
+The error is related to `default_options` be used as dictionary and only can be handled by Conan >= 1.8.
+To fix this error, update Conan package to 1.8 or higher.
```

Comment:
please remove `package` word here as it might be confusing with "conan packages". Use only `Conan` or `Conan client` instead

Reply:
done!

yeah, I thought as "pip package", but wait, Conan be installed by N ways.